Makefile

```diff
@@ -0,0 +1,55 @@
+# type `make help` to see all options 
+
+# If you have naively forked this repo, say to
+# github.com://myname/numpy.org.git, you can test out the build via
+# make TARGET=myname BASEURL=https://myname.github.io/numpy.org deploy
+
+TARGET ?= origin
+BASEURL ?= 
+
+ifdef BASEURL
+	BASEURLARG=-b $(BASEURL)
+endif 
+
+all: build
+
+.PHONY: serve html clean deploy help
+
+.SILENT: # remove this to see the commands executed
+
+serve: public ## serve the website
+	hugo --i18n-warnings server -D
+
+public: ## create a worktree branch in the public directory
+	git worktree add -B gh-pages public $(TARGET)/gh-pages
+	rm -rf public/*
+
+html: public ## build the website in ./public
+	hugo $(BASEURLARG)
+	touch public/.nojekyll
+
+public/.nojekyll: html
```

Comment:
Is this a leftover, or you plan on adding something else?

Reply:
I updated the Makefile in 09f2da9.  I am not certain that it is correct, but it is much closer than what I had.  Do we still want to do all the redirects from the old Moin site?

Reply:
Do we want to add redirects from scipy.org/scipylib/* to scipy.org/* now?  Since this moves everything up a level?

Reply:
Of those, I think only Netlify supports easy redirects.

Reply:
Ah yes, forgot about that. We should probably go with the non-free version then, so we can give anyone who wants to help maintain it access.